### PR TITLE
s390x: Try to show more content of "Show Blocked State" sysrq output

### PR DESCRIPTION
**Pull request by okurz**
The previous approach was stuck on the s3270 state "HOLDING" on one page so
did not show the complete output. It is not sure if this will ensure the
complete content is shown but it looks like at least a bit more is shown :)

Verification run: http://lord.arch/tests/796

### CODE REVIEW

**Comment by okurz** [comment]
@Soulofdestiny

**Comment by okurz** [comment]
I want to check if I can improve the output forwarding with "record_info" even more

**Comment by Soulofdestiny** [comment]
not bad for now =)

File: lib/utils.pm [hunk 1]
```diff
@@ -1154,6 +1156,14 @@ sub reconnect_s390 {
 
     # different behaviour for z/VM and z/KVM
     if (check_var('BACKEND', 's390x')) {
+        my $console = console('x3270');
+        eval { $console->expect_3270(output_delim => 'GNU GRUB'); };
+        if ($@) {
+            diag 'Could not find GRUB screen, continuing nevertheless, trying to boot';
+        }
+        else {
+            $console->sequence_3270("ENTER", "ENTER", "ENTER", "ENTER");
```
Comment (rwx788) [inline lib/utils.pm]: Let's try \n magic here as well and merge if it works.

**Comment by okurz** [comment]
LGTM by @rwx788

**Pull request merged**